File: APP/AndroidScreent/SocketTest/src/com/adsystem/sockettest/NewMessage.java
package com.adsystem.sockettest;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

/**
 * Created by Administrator on 2016/1/14.
 */
public class NewMessage implements Serializable {
    public NewMessage() {

    }

    private MessageTypeEnum MessageType;
    private String SenderIp;
    private int SenderPort;
    private String ReceiverIp;
    private int ReceiverPort;
    private List<String> DataType;
    private List<String> Content;
    private int Second;
    private Date SendTime;
    private String OfficeId;
    private String OperatorId;
    private String Parameter;

    public String getParameter() {
		return Parameter;
	}

	public void setParameter(String parameter) {
		Parameter = parameter;
	}

	public MessageTypeEnum getMessageType() {
        return MessageType;
    }

    public void setMessageType(MessageTypeEnum messageType) {
        MessageType = messageType;
    }

    public String getSenderIp() {
        return SenderIp;
    }

    public void setSenderIp(String senderIp) {
        SenderIp = senderIp;
    }

    public int getSenderPort() {
        return SenderPort;
    }

    public void setSenderPort(int senderPort) {
        SenderPort = senderPort;
    }

    public String getReceiverIp() {
        return ReceiverIp;
    }

    public void setReceiverIp(String receiverIp) {
        ReceiverIp = receiverIp;
    }

    public int getReceiverPort() {
        return ReceiverPort;
    }

    public void setReceiverPort(int receiverPort) {
        ReceiverPort = receiverPort;
    }

    public List<String> getDataType() {
        return DataType;
    }

    public void setDataType(List<String> dataType) {
        DataType = dataType;
    }

    public List<String> getContent() {
        return Content;
    }

    public void setContent(List<String> content) {
        Content = content;
    }

    public int getSecond() {
        return Second;
    }

    public void setSecond(int second) {
        Second = second;
    }

    public Date getSendTime() {
        return SendTime;
    }

    public void setSendTime(Date sendTime) {
        SendTime = sendTime;
    }

    public String getOfficeId() {
        return OfficeId;
    }

    public void setOfficeId(String officeId) {
        OfficeId = officeId;
    }

    public String getOperatorId() {
        return OperatorId;
    }

    public void setOperatorId(String operatorId) {
        OperatorId = operatorId;
    }
}

File: Client/RDTools/RDTools/NewSocketManager/NewMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace RDTools.NewSocketManager
{
    public class NewMessage
    {
        /// <summary>
        /// 消息类型
        /// </summary>
        private MessageTypeEnum messageType;

        public MessageTypeEnum MessageType
        {
            get { return messageType; }
            set { messageType = value; }
        }
        /// <summary>
        /// 终端机器类型
        /// </summary>
        private ComputerEnum computer;

        public ComputerEnum Computer
        {
            get { return computer; }
            set { computer = value; }
        }
        /// <summary>
        /// 发送端IP
        /// </summary>
        private string senderIp;

        public string SenderIp
        {
            get { return senderIp; }
            set { senderIp = value; }
        }
        /// <summary>
        /// 发送端端口
        /// </summary>
        private int senderPort;

        public int SenderPort
        {
            get { return senderPort; }
            set { senderPort = value; }
        }
        /// <summary>
        /// 接收端IP
        /// </summary>
        private string receiverIp;

        public string ReceiverIp
        {
            get { return receiverIp; }
            set { receiverIp = value; }
        }
        /// <summary>
        /// 接收端端口
        /// </summary>
        private int receiverPort;

        public int ReceiverPort
        {
            get { return receiverPort; }
            set { receiverPort = value; }
        }
        /// <summary>
        /// 数据类型
        /// </summary>
        private IList<string> dataType;

        public IList<string> DataType
        {
            get { return dataType; }
            set { dataType = value; }
        }
        /// <summary>
        /// 数据正文
        /// </summary>
        private IList<string> content;

        public IList<string> Content
        {
            get { return content; }
            set { content = value; }
        }


        /// <summary>
        /// 秒数
        /// </summary>
        private int second;

        public int Second
        {
            get { return second; }
            set { second = value; }
        }
        /// <summary>
        /// 发送时间
        /// </summary>
        private DateTime sendTime;

        public DateTime SendTime
        {
            get { return sendTime; }
            set { sendTime = value; }
        }

        private string officeId;

        public string OfficeId
        {
            get { return officeId; }
            set { officeId = value; }
        }

        private string operatorId;

        public string OperatorId
        {
            get { return operatorId; }
            set { operatorId = value; }
        }
    }
}
